hack/test-extended.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+source ${OS_ROOT}/hack/util.sh
+source ${OS_ROOT}/hack/common.sh
+
+# TODO: Randomize these ports
```

Comment:
@bparees @csrwng I'm thinking about making a small Go helper that returns free port (same as we're doing here: https://github.com/openshift/origin/blob/master/test/integration/server_test.go#L44)

(go run test/util/helpers/address.go => 127.0.0.1:RANDOM)